elcid/test/test_fabric.py

```diff
@@ -96,6 +96,7 @@ def test_private_settings_already_exist(self, os, json):
 
 class FabfileTestCase(OpalTestCase):
     def setUp(self):
+        print "starting runing for a test for {}".format(self.__class__)
```

Comment:
Parentheses for print - we should be aiming for python3 compatible where possible I think....

Reply:
Great spot, this should not have been there, removed